Tasks/Maven/mavenutil.ts

```diff
@@ -283,4 +283,20 @@ export function getExecOptions(): tr.IExecOptions {
     return <tr.IExecOptions> {
         env: env,
     };
+}
+
+export function publishMavenInfo(mavenInfo:string) {
+    let stagingDir: string = path.join(tl.getVariable('build.artifactStagingDirectory'), '.mavenInfo');
```

Comment:
Instead of `build.artifactStagingDirectory` can you use `System.DefaultWorkingDirectory`?  Then this will work in release definitions, not just build definitions.  For example, if people want to use Maven to run functional tests in a pre-production environment using a release definition.